docs/docs/live-examples/opentelemetry-store/use-cases/check-shopping-cart-contents.md

```diff
@@ -0,0 +1,94 @@
+# OpenTelemetry Store - Check shopping cart contents
+
+In this use case, we want to validate the following story:
+
+```
+As a consumer
+I want to see my shopping cart
+And see all products that I've added to it
+So I can go to the checkout process
+```
+
+You can trigger this use case by calling the endpoint `GET /api/cart?sessionId={some-uuid}&currecyCode=` from the Frontend service. It should return a payload similar to this:
+```json
+{
+  "userId": "8c0465e2-32bb-4ecb-a9c8-5a2861629ff1",
+  "items": [
+    {
+      "productId": "66VCHSJNUP",
+      "quantity" : 1,
+      "product": {
+        "id": "66VCHSJNUP",
+        "name": "Starsense Explorer Refractor Telescope",
+        "description": "The first telescope that uses your smartphone to analyze the night sky and calculate its position in real time. StarSense Explorer is ideal for beginners thanks to the app’s user-friendly interface and detailed tutorials. It’s like having your own personal tour guide of the night sky",
+        "picture": "/images/products/StarsenseExplorer.jpg",
+        "priceUsd": {
+          "currencyCode": "USD",
+          "units": 349,
+          "nanos": 950000000
+        },
+        "categories": [
+          "telescopes"
+        ]
+      }
+    }
+  ]
+}
+```
+
+If it is the first time that you are calling this endpoint, to see an item into the shopping cart you need first to [Add item into shopping cart](./add-item-into-shopping-cart.md).
```

Comment:
This looks like a good spot to introduce `transactions`. Does it make sense to introduce them here?

Reply:
Sure! I'm thinking on create another page with a transaction with 3 tests: add item to cart -> check cart -> checkout